src/wovnio/wovnphp/Utils.php

```diff
@@ -105,6 +105,18 @@ public static function isIgnoredPath($uri, $store)
         );
     }
 
+    public static function shouldIgnoreBySitePrefixPath($uri, $store)
+    {
+        if ($store &&
```

Comment:
This seems overly defensive. When will store be null? If there is a case, I would prefer we do an early return in wovn_interceptor and assume everywhere else that store is non-null. Headers.requestOut already assumes it is non-null.

Can we also assume that Store will initialize `site_prefix_path` and `url_pattern_name` keys?

Reply:
By the way, it seems strange to me that we do `headers->requestOut` before we check if the request is ignored.

Reply:
> Can we also assume that Store will initialize site_prefix_path and url_pattern_name keys?

We initialize keys. So, We can assume there are `site_prefix_path` and `url_pattern_name` keys.

> By the way, it seems strange to me that we do headers->requestOut before we check if the request is ignored.

Ah, yes. This code means that WOVN will remove lang code even if the REQUEST_URI is ignored.

Reply:
> By the way, it seems strange to me that we do headers->requestOut before we check if the request is ignored.

I will split PR with more tests to change this.